pymc/distributions/timeseries.py

```diff
@@ -245,6 +274,7 @@ def dist(
                 and init.owner.op.ndim_supp == 0
             ):
                 raise TypeError("init must be a univariate distribution variable")
+            check_dist_not_registered(init)
```

Comment:
Can you add a comment what this line does?

Reply:
I think the function self documents itself well enough: https://github.com/pymc-devs/pymc/blob/02897d159de9eec12951595cdc5869791f2e7f93/pymc/util.py#L337-L351